jsk_perception/cfg/TemplateMatchDetector.cfg

```diff
@@ -0,0 +1,22 @@
+#! /usr/bin/env python
+
+PACKAGE = 'jsk_perception'
+ID = "TemplateMatchDetector"
+
+from dynamic_reconfigure.parameter_generator_catkin import *
+
+gen = ParameterGenerator()
+
+gen.add("template_name", str_t, 0, "template image name", "template.png")
+gen.add("min_scale", double_t, 0, "min_scale of template", 0.4, 0.0, 1.0)
+gen.add("max_scale", double_t, 0, "max_scale of template", 1.6, 1.0, 10.0)
+gen.add("resize_num", int_t, 0, "the number of resizing template", 20, 5, 50)
+gen.add("matching_thre", double_t, 0, "matching_threshold", 0.7, 0.0, 1.0)
+gen.add("target_num", int_t, 0, "the number of target", 6, 1, 10)
+gen.add("max_detect_num", int_t, 0, "if detected rectangle number is over max_detect_num, skip calculation", 15000, 2000, 30000)
```

Comment:
Why default, min_value and max_value are so large?
Maybe `max_detection_num` is better?